src/applications/disability-benefits/wizard/pages/disagreeing.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import moment from 'moment';
+import ErrorableDate from '@department-of-veterans-affairs/formation-react/ErrorableDate';
+import { pageNames } from './pageList';
+
+// Figure out which page to go to based on the date entered
+const findNextPage = state => {
+  const date = moment({
+    day: state.day.value,
+    // moment takes 0-indexed months, but the date picker provides 1-indexed months
+    month: parseInt(state.month.value, 10) - 1,
+    year: state.year.value,
+  });
+  const feb19 = moment('2019-02-19');
+
```

Comment:
Saw this magic date in the related issue; just double checking that we won’t need to change this ever?

We may also want to leave a comment for anybody else looking at this in the future

Reply:
Good call. I think it's pretty much set in stone; I'll drop a comment in there for clarification.